index.html

```diff
@@ -14,6 +17,121 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+
+    <header>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#intro">About</a>
+        <a href="https://ifiokudoidiok.github.io/portfolio/">Portfolio</a>
+        <a href="#contact">Contact</a>
+      </nav>
+    </header>
+    <main>
+      <div class="top-section">
+        <div class="image">
+          <img src="./img/user-interface.jpg" alt="my-img">
+        </div>
+
+        <div class="top-text">
+          <h1 class="intro" id="intro"> Ifiok Udoidiok - Software Engineer</h1>
+
+          <p>Technology is my jam!</p>
+          <br>
+          <p>
+            Computer Science- Fullstack Web Development Track at Lambda School || Freelance Software Developer at Fiverr
+            US</p>
+
+        </div>
+
+      </div>
+
+      <div class="mid-section">
+        <div class="my-summary" id="summary">
+          <h1>Summary...</h1>
+          <p>I am:
+            1. A Front-end Javascript Developer with 3years+ Experience in web development using Vanilla JS.
```

Comment:
You could add line breaks <br/> to start each of these numbered elements on a new line. Alternatively, why not use ordered list instead **<ol><li></li></ol>**